x/stakeibc/keeper/icqcallbacks_pool_price.go

```diff
@@ -55,19 +54,19 @@ func PoolPriceCallback(k Keeper, ctx sdk.Context, args []byte, query icqtypes.Qu
 	// Get the associate "SpotPrice" from the twap record, based on the asset ordering
 	// The "SpotPrice" is actually a ratio of the assets in the pool
 	var price sdk.Dec
-	if twapRecord.Asset0Denom == tradeRoute.TargetDenomOnTradeZone {
+	if twapRecord.Asset0Denom == tradeRoute.HostDenomOnTradeZone {
 		price = twapRecord.P0LastSpotPrice
 	} else {
 		price = twapRecord.P1LastSpotPrice
 	}
 
 	k.Logger(ctx).Info(utils.LogICQCallbackWithHostZone(chainId, ICQCallbackID_PoolPrice,
 		"Query response - price ratio of %s to %s is %s",
-		tradeRoute.RewardDenomOnTradeZone, tradeRoute.TargetDenomOnTradeZone, price))
+		tradeRoute.RewardDenomOnTradeZone, tradeRoute.HostDenomOnTradeZone, price))
 
 	// Update the price and time on the trade route data
 	tradeRoute.SwapPrice = price
-	tradeRoute.PriceUpdateTime = ctx.BlockTime()
+	tradeRoute.PriceUpdateTimestamp = ctx.BlockTime().Unix()
```

Comment:
Is this nanos, I thought it was seconds? Should it be nanos everywhere?

Reply:
ok I see in the protos this was intended to be seconds, so I think this is correct as intended but a lot of other places in the code base that we use timestamps (like the timestamps passed into the ICA msgs) are nanos

Reply:
Also the epoch trackers are nanos so like strideEpochTracker.NextEpochStartTime is nanos I believe

Reply:
yeah a lot of other stuff is in nanos, but this shouldn't interact with anything else, so I didn't think it was necessary to make them consistent

i don't have a strong preference, I can switch it to nanos if you prefer

Reply:
sg, I'll do nano. 

int64 was just so there's no need to cast. I can do uint for that as well if you think it makes sense